scripts/aws/lambda/TM-Extractor/_envcommon/lambda.hcl

```diff
@@ -0,0 +1,46 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# COMMON TERRAGRUNT CONFIGURATION
+# This is the common component configuration for mysql. The common variables for each environment to
+# deploy mysql are defined here. This configuration will be merged into the environment configuration
+# via an include block.
+# ---------------------------------------------------------------------------------------------------------------------
+
+# Terragrunt will copy the Terraform configurations specified by the source parameter, along with any files in the
+# working directory, into a temporary folder, and execute your Terraform commands in that folder. If any environment
+# needs to deploy a different module version, it should redefine this block with a different ref to override the
+# deployed version.
+terraform {
+  # Sourcing from here rn, updating to a release from https://github.com/hotosm/TM-Extractor/ later.
```

Comment:
In order to make my tests work, I had to replace the source with my local build of TM-extractor where I set the required terraform version to `~>1.6.0`. See https://github.com/hotosm/TM-Extractor/issues/3 for more details.

Reply:
Yep, that would be nice to have minor patches allowed, only thing is that once we have our CircleCi config for deployments how would we make sure they align. Else i am all okay. 
I have updated [TM-Extractor](https://github.com/hotosm/TM-Extractor/tags)'s v.1.0.1 to be source. I see it has updated code on `=1.6.0`.